src/creds.jl

```diff
@@ -36,3 +48,16 @@ mutable struct ClientCredentials <: Credentials
     ClientCredentials(client_id, client_secret, client_credentials_url = nothing) =
         new(client_id, client_secret, client_credentials_url, nothing)
 end
+
+function Base.show(io::IO, c::ClientCredentials)
+    print(
+        io,
+        "(",
+        c.client_id,
+        c.client_secret == nothing ? "" : ", $(c.client_secret[1:3])...",
+        c.access_token == nothing ? "" : ", $(c.access_token)",
```

Comment:
Shouldn't this also have the [1:3]?

Reply:
`access_token` is a struct and it's handled in a different show function `function Base.show(io::IO, t::AccessToken)`